recorder.cpp

```diff
@@ -107,20 +122,29 @@ struct RecorderImplementation : public Recorder {
 
     void init() {
         output.precision(10);
+        jsonlProcessor = accelerated::Processor::createThreadPool(1);
+        #ifdef USE_OPENCV_VIDEO_RECORDING
+        frameStore = std::make_unique<recorder::Allocator<cv::Mat>>(
```

Comment:
Maybe it would be good to set the initial and max. capacity here explicitly (maybe just something like `constexpr size_t MAX_CAPACITY = 20;` before this line). The defaults may be a bit random and this affects, e.g., the memory consumption significantly